src/components/IssueController.jsx

```diff
@@ -61,7 +61,7 @@ export default class IssueController extends FalcorController {
           //console.log("Category: " + JSON.stringify(category));
           return(
             <div key={category} className="issue__category">
-              <h2 className="section-header">{category}</h2>
+              <h2 className="section-header">{category.replace('-', ' ')}</h2>
```

Comment:
Yeah the fetch itself wouldn't cost anything because it would all be
cached.

On Sun, Aug 7, 2016, 4:25 PM Emil Goldsmith Olesen <notifications@github.com>
wrote:

> In src/components/IssueController.jsx
> <https://github.com/thegazelle-ad/gazelle-front-end/pull/41#discussion_r73802822>
> :
>
> > @@ -61,7 +61,7 @@ export default class IssueController extends FalcorController {
> >            //console.log("Category: " + JSON.stringify(category));
> >            return(
> >              <div key={category} className="issue__category">
> > -              <h2 className="section-header">{category}</h2>
> > +              <h2 className="section-header">{category.replace('-', ' ')}</h2>
>
> I don't think it's much extra work, it'll be within the same query.
> Personally I think it would be more than okay having a category slug field,
> where we get data is from Falcor, so that's where we should keep on getting
> it.
>
> If we really want to optimize it and make all slugs just be a replace
> spaces with hyphens, we can also make Falcor do that itself, and not fetch
> anything extra. Just write it into the Falcor path that if you fetch name
> and slug at the same time, return slug as a transform of name.
>
> As of right now I did also add a slug field to the category table in the
> database though, so if we keep that, we should of course also use it. And
> that gives more flexibility in choosing slug names than just the transform.
> We can of course just decide it's always a transform, that's a design
> decision I guess.
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/thegazelle-ad/gazelle-front-end/pull/41/files/a85a19c83b836058745a11879e25ef14a8a7a67e#r73802822>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADAFzaI5fSa9QSzmmgwVX7f6yf8ClQQDks5qderTgaJpZM4JdXZj>
> .
>